ucp/_libs/ucx_worker.pyx

```diff
@@ -120,7 +120,7 @@ cdef class UCXWorker(UCXObject):
         status = ucp_worker_create(context._handle, &worker_params, &self._handle)
         assert_ucs_status(status)
         self._inflight_msgs = set()
-        self._inflight_msgs_to_cancel = set()
+        self._inflight_msgs_to_cancel = dict({"tag": set()})
```

Comment:
```suggestion
        self._inflight_msgs_to_cancel = {"tag": set()}
```

Reply:
Do we need to use `dict` here explicitly?

Reply:
You're right, that's redundant, applied the suggestion now.